src/main/java/org/fenixedu/academic/report/academicAdministrativeOffice/RegistryDiploma.java

```diff
@@ -103,7 +103,8 @@ protected void setHeader() {
 
         if (!graduationLevel.isEmpty()) {
             degreeRegistrationDiploma =
-                    Joiner.on(" ").join(degreeRegistrationDiploma, BundleUtil.getString(Bundle.ACADEMIC, "label.of.both"),
+                    Joiner.on(" ").join(degreeRegistrationDiploma,
+                            BundleUtil.getLocalizedString(Bundle.ACADEMIC, "label.of.both").getContent(getLocale()),
```

Comment:
Can't you just invoke the `getString` variant with a specific locale? Or do you really need the Localized String locale resolution?